tests/integration/routes/profile/patch-profile-test.js

```diff
@@ -15,9 +15,9 @@ var routeOptions = {
   },
   payload: {
     data: {
-      id: 'userid123-profile',
       type: 'profile',
       attributes: {
+        id: 'userid123-profile',
```

Comment:
why did you move the id? It should be next to `type` where it was, it’s part of the JSON API spec